concepts/numbers/introduction.md

```diff
@@ -0,0 +1,18 @@
+# Haskell Built-in Numeric Types
+
+Integers` hold any number no matter how big, up to the limit of your machine's memory, while `Int` corresponds to the set of positive and negative integers that can be expressed in 32 or 64 bits (guaranteed at least -2^29 to 2^29).
```

Comment:
If I look at the exemplar solution, it looks like only `Integer` is used, but not `Int`. In principle, this means that the section on `Int` should be removed, but I think in this particular case it is fine to keep it in.